lib/command.js

```diff
@@ -1,38 +1,14 @@
 'use strict';
 
-var program = require ('commander');
-
+var common = require ('./common');
+var program = require ('./program');
 var doctest = require ('..');
-var pkg = require ('../package.json');
-
-
-program
-.version (pkg.version)
-.usage ('[options] path/to/js/or/coffee/module')
-.option ('-m, --module <type>',
-         'specify module system ("amd" or "commonjs")')
-.option ('    --nodejs <options>',
-         'pass options directly to the "node" binary')
-.option ('    --prefix <prefix>',
-         'specify Transcribe-style prefix (e.g. ".")')
-.option ('    --opening-delimiter <delimiter>',
-         'specify line preceding doctest block (e.g. "```javascript")')
-.option ('    --closing-delimiter <delimiter>',
-         'specify line following doctest block (e.g. "```")')
-.option ('-p, --print',
-         'output the rewritten source without running tests')
-.option ('-s, --silent',
-         'suppress output')
-.option ('-t, --type <type>',
-         'specify file type ("coffee" or "js")')
-.parse (process.argv);
 
-//  formatErrors :: Array String -> String
-function formatErrors(errors) {
-  return (errors.map (function(s) { return 'error: ' + s + '\n'; })).join ('');
-}
 
 var errors = [];
+if (program.module === 'esm' || program.type === 'esm') {
```

Comment:
This is where people end up when trying to run `/bin/doctest --module esm` with Node < 9 on their system.